File: __tests__/__fixtures__/output/javascript/axios/headers.js
import axios from "axios";

const options = {
  method: 'GET',
  url: 'https://httpbin.org/headers',
  headers: {accept: 'text/json', 'x-foo': 'Bar', 'x-bar': 'Foo'}
};

axios.request(options).then(function (response) {
  console.log(response.data);
}).catch(function (error) {
  console.error(error);
});

File: src/targets/csharp/restsharp/fixtures/headers.cs
using RestSharp;


var options = new RestClientOptions("https://httpbin.org/headers");
var client = new RestClient(options);
var request = new RestRequest("");
request.AddHeader("accept", "application/json");
request.AddHeader("x-foo", "Bar");
request.AddHeader("x-bar", "Foo");
var response = await client.GetAsync(request); 